### ISSUE TITLE
Parse/format example using wrong React/ReactDOM versions

### ISSUE THREAD

**Issue by ntocampos** [final-form/react-final-form#421]
### Are you submitting a **bug report** or a **feature request**?
Bug report

### What is the current behavior?
The `parse`/`format` sandbox example isn't working due to its React/ReactDOM versions. This example is currently using version `16.0.0`. In this one, `React.createContext()` isn't available yet, and that's what's breaking the example.

### What is the expected behavior?
The new context API is available from version `16.3.0`. Updating both React and React DOM versions in the `package.json` file should do it.

### Sandbox Link
https://codesandbox.io/s/10rzowm323

### What's your environment?
```
"dependencies": {
    "styled-components": "latest",
    "react-final-form": "latest",
    "react-dom": "16.0.0",
    "react": "16.0.0",
    "final-form": "latest"
  }
```

### Other information
I would fix this small bug, but I didn't figure it out how to edit the sandbox code. If there's a way, I'd gladly do it.

**Comment by aikidoshi** [comment]
I tried downloading the demo, and after upgrading to the latest react version to get it to work I now get this warning:
_A component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
    in input (created by Field)
    in Field (created by Context.Consumer)
    in _class (at src/index.js:39)
    in div (at src/index.js:37)
    in form (at src/index.js:36)
    in ReactFinalForm (at src/index.js:32)
    in div (created by Context.Consumer)
    in StyledComponent (created by styled.div)
    in styled.div (at src/index.js:26)
    in App (at src/index.js:77)_

**Comment by ntocampos** [comment]
That's odd 🤔 
This isn't happening at the sandbox. Did you upgrade both `react` and `react-dom` versions?

**Comment by ntocampos** [comment]
That's true. Didn't noticed that. I don't know much about the lib's internal workings, but it only happens when the `parse` and `format` props are defined 🤔
